modules/displayBook.js

```diff
@@ -0,0 +1,19 @@
+import deleteFunc from './delete.js';// eslint-disable-line
+import myLib from '../index.js';// eslint-disable-line
+
+const bookContainer = document.querySelector('.books');
+export default function displayBook() {
+  bookContainer.innerHTML = '';
+  myLib.forEach((s) => {
+    bookContainer.innerHTML += ` 
+                <div class="book-item">
+                    <p class="title"> "${s.title}" by ${s.author}</p> 
+                    <button class="remove">Remove</button>
+                </div>
```

Comment:
- [ ] Same here, Please, it's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions, Kindly make sure to address the requested changes :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)